perf/lua/column_insert.lua

```diff
@@ -18,6 +18,10 @@ local benchmark = require('benchmark')
 local USAGE = [[
    engine <string, 'memtx'>          - space engine to use for the test
    wal_mode <string, 'write'>        - write-ahead log mode to use for the test
+   sparse_mode <string, 'rand'>      -
```

Comment:
Just to let you know:
It is desired to make sure the benchmark is stable enough with new changes before merging it to the master branch. `perf_micro` workflow executes benchmarks on regular runners if PR is based on a branch created in a fork repository, and executes on a special hardware machine if PR is based on a branch created in a main repository. You can push the same branch to the main repo and create janitorial PR for review results in https://mon.tarantool.io/.

Reply:
> There are no results for branch iverbin/perf-sparse_mode-column_insert-lua at https://mon.tarantool.io/.

I don't know why they didn't show up there.

> How did you compare results with master?

There is no sense for this patch. It switched the test mode, so it can be considered as a different benchmark.

Reply:
> I don't know why they didn't show up there.

Probably because you execute `perf_micro` workflow from a PR based on a branch in a forked repository, instead of PR based on a branch in the main repository

> so it can be considered as a different benchmark.

Then it makes sense to confirm that bench produce stable results on each run.

Reply:
Looks stable:

<img width="992" alt="Screenshot 2024-12-13 at 22 28 29" src="https://github.com/user-attachments/assets/76ed3f2c-a49a-46ab-a4a7-0c89f3112868" />

https://mon.tarantool.io/grafana/d/olYWJGsSk/performance-perf_micro?orgId=1&var-bucket=perf-debug&var-measurement=column_insert&var-branch=iverbin%2Fperf-sparse_mode-column_insert-lua&var-subtest=All&from=now-6h&to=now